README.md

```diff
@@ -46,22 +54,38 @@ kubectl apply -f fybrikapplication.yaml
 Run the following command to wait until the fybrikapplication be ready.
 ```bash
 while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
+
+while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.assetStates.fybrik-notebook-sample/iceberg-dataset.conditions[?(@.type == "Ready")].status}') != "True" ]]; do echo "waiting for fybrik-notebook-sample/iceberg-dataset asset" && sleep 5; done
 ```
 
 Wait For the pod `my-notebook-default-trino-module-xxxx` to be completed. This pod runs a python code that registers the asset in trino and applies the policy to create a virtual dataset. The user can use the following username to connect to trino:
 
     "name": "user1"
 
-For example, you can run trino docker container and run queries. Run the following command to run trino server.
+For example, you can run trino docker container and run queries. First, check the docker container name of trino (the docker container with the image `trinodb/trino:latest`). Then, Run the following command to run trino server.
 ```bash
-docker container exec -it trinoicebergminio_trino-coordinator_1 trino --user user1
+docker container exec -it <trino_container_name> trino --user user1
 ```
 Check the tables that `user1` can see. It should be only the `view1`.
 ```bash
 show tables from iceberg.icebergtrino;
 ```
 
-You can run a query to select from the created view. I should return only allowed columns according to the policies.
+You can run a query to select from the created view. It should return only allowed columns according to the policies.
 ```bash
 select * from iceberg.icebergtrino.view1;
-```
\ No newline at end of file
+```
```

Comment:
Please add that  in the output we don't see the columns with PII data (a, d)

we can add that if we login into the trino as admin, we can see both table "logs" and view view1`